testing/src/main/java/org/motechproject/nms/testing/web/TestingController.java

```diff
@@ -62,15 +62,21 @@ public String createSubscriptionPacks() {
 
     @RequestMapping(value = "/createMctsMoms")
     @ResponseBody
-    public String createMctsMoms(@RequestParam(required = true) int count) throws IOException {
-        return testingService.createMctsMoms(count);
+    public String createMctsMoms(
+            @RequestParam(required = true) int count,
```

Comment:
right, and looking at data.py from above - 
`if args.mctsmoms > 0:
-        exec_http_get("{}/module/testing/createMctsMoms".format(args.server), {'count': args.mctsmoms})		+        params = {'count': args.mctsmoms, 'lmp': args.lmp > 0}
+        exec_http_get("{}/module/testing/createMctsMoms".format(args.server), params)`

How does this work? What kind of trickery is this? :confounded: